u03_proteus.xml

```diff
@@ -1,15 +1,15 @@
 <div type="episode" n="03">
-<p><lb n="030001"/>Ineluctable modality of the visible: at least that if no more, thought
-<lb n="030002"/>through my eyes. Signatures of all things I am here to read, seaspawn and
-<lb n="030003"/>seawrack, the nearing tide, that rusty boot. Snotgreen, bluesilver, rust:
-<lb n="030004"/>coloured signs. Limits of the diaphane. But he adds: in bodies. Then he was
+<p><lb n="030001"/><said who="sd" aloud="false">Ineluctable modality of the visible: at least that if no more, thought
+<lb n="030002"/>through my eyes. Signatures of all things I am here to read, <distinct type="nonstandard-compound">seaspawn</distinct> and
+<lb n="030003"/><distinct type="nonstandard-compound">seawrack</distinct>, the nearing tide, that rusty boot. <distinct type="nonstandard-compound">Snotgreen</distinct>, <distinct type="nonstandard-compound">bluesilver,</distinct> rust:
+<lb n="030004"/>coloured signs. Limits of the <distinct type="archaism">diaphane.</distinct> But he adds: in bodies. Then he was
 <lb n="030005"/>aware of them bodies before of them coloured. How? By knocking his
 <lb n="030006"/>sconce against them, sure. Go easy. Bald he was and a millionaire, <quote xml:lang="it">maestro
-<lb n="030007"/>di color che sanno</quote>. Limit of the diaphane in. Why in? Diaphane,
-<lb n="030008"/>adiaphane. If you can put your five fingers through it it is a gate, if not a
-<lb n="030009"/>door. Shut your eyes and see.</p>
+<lb n="030007"/>di color che sanno</quote>. Limit of the <distinct type="archaism">diaphane</distinct> in. Why in? <distinct type="archaism">Diaphane,</distinct>

```

Comment:
I'm not sure this is an archaism, necessarily, but rather just a term from Greek philosophy that might not be very common, like "ethos." I'd recommend maybe changing this to `<foreign>`, instead.